components/skus/renderer/skus_js_handler.cc

```diff
@@ -194,10 +194,13 @@ void SkusJSHandler::OnFetchOrderCredentials(
                                  v8::MicrotasksScope::kDoNotRunMicrotasks);
 
   v8::Local<v8::Promise::Resolver> resolver = promise_resolver.Get(isolate);
-  v8::Local<v8::String> result;
-  result = v8::String::NewFromUtf8(isolate, response.c_str()).ToLocalChecked();
 
-  std::ignore = resolver->Resolve(context, result);
+  // Create an empty string as the result
+  v8::Local<v8::String> empty_result =
+      v8::String::NewFromUtf8(isolate, "").ToLocalChecked();
+
+  // Always resolve the promise with the empty string
+  std::ignore = resolver->Resolve(context, empty_result);
```

Comment:
Temporarily making this always resolve to the empty string in order to make it easier to debug why the promise does not resolve.